source/Wormhole.cpp

```diff
@@ -33,6 +33,7 @@ void Wormhole::Load(const DataNode &node)
 		return;
 	isDefined = true;
 	isAutogenerated = false;
+	linkColor = ExclusiveItem<Color>(GameData::Colors().Get("map wormhole"));
```

Comment:
You should probably do this at the end of the Load method, and only if the linkColor object is empty.
As it is, if the `Load` method in this `Wormhole` is called a second time, say because a plugin wants to edit some part of it, the link colour will be set to the default even if the second Load call didn't want to change it.

Reply:
@Azure3141 I've opened a PR to your branch (again) that does what I suggested on Discord.
[thing](https://github.com/Azure3141/endless-sky/pull/5)